.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: 🐛 Bug Report
 about: Create an issue report.
+labels: 'status: have to reproduce, type: bug 🐛'
```

Comment:
This label can be not correct because of the interpretation / usage of Strapi.
For this kind of issue, I like to have **no labels** because we can see issues that have not be managed or at least readen by one of the team.

So if you can remove this line I will appreciate.

Reply:
Fixed @lauriejim